docs/references/configuration.rst

```diff
@@ -628,7 +628,12 @@ jwt-role-claim-key
     # the DSL accepts characters that are alphanumerical or one of "_$@" as keys
     jwt-role-claim-key = ".postgrest.roles[1]"
 
-    # {"https://www.example.com/role": { "key": "author }}
+    # {"postgrest":{"roles": ["other", "author"]}}
+    # accepts string comparison operators, supported operators are "==", "!=", "^==", "$==", "*=="
```

Comment:
These look sufficient to solve all the cases on https://github.com/PostgREST/postgrest/issues/1536.